azure-pipelines.yml

```diff
@@ -97,4 +105,37 @@ jobs:
       brew tap LouisBrunner/valgrind
       brew install --HEAD LouisBrunner/valgrind/valgrind
       make TARGET=SANDYBRIDGE NO_AVX512=1 USE_OPENMP=1 INTERFACE64=1 DYNAMIC_ARCH=1 DYNAMIC_LIST=SANDYBRIDGE DEBUG=1 NO_PARALLEL_MAKE=1 CC=/usr/local/opt/llvm/bin/clang FC=gfortran-10
-      cd ctest; OMP_NUM_THREADS=1 valgrind ./xscblat2 <sin2
```

Comment:
I had been wondering why the valgrind had remained... That means it should also be possible to remove the installation of it...

Reply:
yes, I forgot to remove it - will drop that (and return the build to full DYNAMIC_ARCH) when I have the ifort build figured out.